app/docs/0.14.x/configuration.md

```diff
@@ -607,21 +607,29 @@ Default: `60`
 
 ---
 
-##### **server_tokens**
-
-Enables or disables emitting Kong version on error pages and in the `Server`
-or `Via` (in case the request was proxied) response header field.
-
-Default: `on`
-
----
-
-##### **latency_tokens**
-
-Enables or disables emitting Kong latency information in the `X-Kong-Proxy-Latency`
-and `X-Kong-Upstream-Latency` response header fields.
-
-Default: `on`
+##### **headers**
+
+Comma-separated list of headers which should be injected by Kong.
+
+Values can be:
+  response header fields.
```

Comment:
This phrase could be rephrased to be more exact, and its formatting is a bit weird. This is my suggestion:

> Each value on the list can be one of the response header names below, or one of the two accepted aliases(`server_tokens` or `latency_tokens`).

Also, please include a blank line between this paragraph and the list following it. Some markdown libraries don't parse that as a list without it.